_posts/2016-03-04-terrier-open-source-sensitive-democracy-for-hamilton-tles.md

```diff
@@ -0,0 +1,90 @@
+author: cecily
+layout: blog-post-tgif
+comments: true
+---
+
+TGIF! – Thank Grohl, Gauss, Glob, Galileo and Science it’s Friday, and again it's time for our weekly reading recommendations, prepared to help you get some good reading time on your weekend. They're curated, just for you, by [Cecily](http://twitter.com/skeskali).
+
+## Open Source
+
+> NodeTogether is an initiative to improve the diversity of the Node community by bringing people of underrepresented groups together to learn Node.js
+<cite>[Node Together](http://www.nodetogether.org/)</cite>
+
+[Microsoft Finally Loves Open Source: A Chat with npm's Founder](http://thenextweb.com/insider/2016/03/03/microsoft-finally-s-open-source-a-chat-with-npms-founder/)
+
+[Healthy Open Source](https://medium.com/the-javascript-collection/healthy-open-source-967fa8be7951#.6jb6j68t7)
+
+[The 2016 Annual Apache CouchDB User Survey](http://blog.couchdb.org/2016/03/01/the-2016-annual-apache-couchdb-user-survery/)
+
+[Terrier, the open source tool used to retrieve metadata from scholarly sources](https://thewinnower.com/posts/terrier-a-open-source-tool-used-to-retrieve-metadata-of-scholarly-works-from-a-variety-of-sources)
+
+
+
+## Design
+
+> I’ve been paying close attention to these small breaks in UX that some people seem so happy to write off — the misplaced humor, the snotty error messages, the binary gender selection boxes, the assumptions about sexuality, the inability to select more than one race. What I’ve realized is that the best way to look at them is as a series of microaggressions.
+
+>Lots of people think caring about these microaggressions is a waste of time, too: Stop being so sensitive! There are more important issues in the world! Everyone’s a victim these days!
+
+> Mostly the people who don’t experience them.
+
+> For those who do, they’re exhausting. They stack up over the course of a day, a week at work, a lifetime of being fed the same BS. A million tiny moments where you were reminded that you don’t fit in, or that you’re not “normal,” or that others view your body as fair game.
+
+<cite>Sara Wachter-Boettcher, [Too Sensitive](https://medium.com/@sara_ann_marie/too-sensitive-9752a86a8382#.s4yig43qu)</cite>
+
+>I have a rule: If I keep complaining about something, I either do something about it or let it go.
+ <cite> Tina Roth-Eisenberg, [Don’t Complain, Create](http://99u.com/videos/28625/tina-roth-eisenberg-5-rules-for-making-an-impact "Don't Complain, Create")</cite>
+
+[Mosaic Tiles That Resemble African Fabrics](http://design-milk.com/mosaic-tiles-resemble-african-fabrics/)
+
+[How the Heck Do You Hire A Web Design Agency?](https://css-tricks.com/how-the-heck-do-you-hire-a-web-design-agency/)
+
+## The Web and Development
+
+[Preload: What Is It Good For?](https://www.smashingmagazine.com/2016/02/preload-what-is-it-good-for/)
+
+## Events
+If you're in Washington, DC, come [Meet The Hoodies](https://ti.to/hoodie/dc-meet-the-hoodies) on Sunday, March 6.
```

Comment:
Can we add [Intro to Open Source](http://www.meetup.com/Jr-Dev-Mentoring/events/228951389/),  April 2, Atlanta. I’ll co-host with [Kim](https://twitter.com/KimCrayton1/)